src/effects/builtin/metronomeeffect.cpp

```diff
@@ -89,9 +89,11 @@ void MetronomeEffect::processChannel(
 
     unsigned int maxFrames;
     if (m_pSyncParameter->toBool() && groupFeatures.has_beat_length_sec) {
-        maxFrames = bufferParameters.sampleRate() * groupFeatures.beat_length_sec;
+        maxFrames = static_cast<unsigned int>(
```

Comment:
General: Use `static_cast<decltype(maxFrames)>` instead of repeating types.

https://en.cppreference.com/w/cpp/language/decltype

Reply:
But doesn't this make harder to see what type is being cast to? In this case it's probably not that important because the type is declared 2 lines above. 

Also, doesn't this inhibit a compiler warning if I change the type later on?  E.g. if I have a double value and change it to float later, the compiler won't warn about the data loss?

Reply:
The intention is to cast the expression on the right-hand side to the type of the variable on the left-hand side. Why not explicitly state that? What's the advantage of repeating the type literally instead?